File: 3392-count-subarrays-of-length-three-with-a-condition/3392-count-subarrays-of-length-three-with-a-condition.cs
public class Solution {
    public int CountSubarrays(int[] nums) {
        var count = 0;
        for(int l=0, m=1, r=2; r < nums.Length; l++, m++, r++)
            if(nums[m] == 2 * (nums[l] + nums[r]))
                count++;
        return count;
    }
}

File: 3392-count-subarrays-of-length-three-with-a-condition/3392-count-subarrays-of-length-three-with-a-condition.py
class Solution:
    def countSubarrays(self, nums: List[int]) -> int:
        count = 0
        for m in range(1, len(nums) - 1):
            if nums[m] / 2 == nums[m-1] + nums[m+1]:
                count += 1
        return count